client/main.lua

```diff
@@ -4,6 +4,12 @@ local volume = 0.3
 local intialized = false
 local voiceTarget = 1
 
+-- radio mix
```

Comment:
This would probably better by line 40 so its easier to find.

Reply:
?

Reply:
Move all of the radio submix stuff to where the radio is toggled

Reply:
sure